frontend/components/Dialog/ConfigView.tsx

```diff
@@ -105,7 +106,7 @@ function BasicTabs({ onClose }: BasicTabsProps) {
       filters: [{ name: 'db', extensions: ['db'] }],
     };
     const setPathCallback = (val) => setPath({ path: val });
-    menuDispatch({
+    dispatch({
```

Comment:
Replace current Context API code with correct redux function calls. You will need to import those functions from the slices